src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -139,6 +149,16 @@ function assertValidProps(component, props) {
     'not a string. For example, style={{marginRight: spacing + \'em\'}} when ' +
     'using JSX.'
   );
+  if (__DEV__) {
+    Object.keys(props).forEach(function(propName) {
+      warning(
+        !eventTypes.hasOwnProperty(propName),
```

Comment:
This would for example not catch `onMouseup`. Maybe something like the following would be better?
```js
var eventType = eventTypes[propName.toLowerCase()];
warning(
  !eventType || eventType === propName,
```